### DISCUSSION TITLE
Login error, plug-in site unavailable

### DISCUSSION THREAD

**Discussion by ivyhi11** [homebridge-simplisafe3/homebridge-simplisafe3#242] [Q&A]
Getting no response in the home app and logs showing login error and plug-in site unavailable. What does this mean and how can I resolve it? Updated to iOS 15 yesterday and it started doing this this morning…

**Comment by shamoon** [comment]
What is "plug-in site"? Doubt this is an issue with this specific plug-in, sounds like your homebridge install. Restarting everything usually helps...
